src/components/Button/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { PropsWithChildren, MouseEvent } from 'react';
+import { styButtonPrimary, styUndefinedBtn } from './styles';
+import { ButtonProps } from './buttonProps';
+import { isValidHttpUrl, Route } from './Route';
+import { cx } from '@emotion/css';
+
+const Button = (props: PropsWithChildren<ButtonProps>): JSX.Element => {
+    const { btnSize, btntype, btnColor, link, element, children, ...rest } =
+        props;
+    const className =
+        btnSize !== undefined
+            ? `${btnSize} ${styButtonPrimary}`
+            : cx(styButtonPrimary, styUndefinedBtn);
+    const color = { backgroundColor: btnColor };
+
+    const handleClick = (e: MouseEvent): void => {
+        if (link !== undefined) {
+            if (element !== undefined) {
```

Comment:
```suggestion
        if (link) {
            if (element) {
```

kayaknya bisa gini deh.. atau ini kemarin kita dah sempet coba?